### PR TITLE
Week2 Assignments

### PR DESCRIPTION
**Pull request by PranayKudale**
Session 1 assignments

### CODE REVIEW

**Comment by PranayKudale** [comment]
Session 2 assignments

**Comment by sairamin** [comment]
Here are few suggestions you need to incorporate into your PR's:

- Neat and crisp title
- Clear description about what is the problem and what to expect in the PR review
- Consistent naming convention (Please read about camelCase, PascalCase, snake_case) and follow one (preferably camelCase)

File: Week-1/Session2/Hometownwebpage.html
Comment (sairamin) [inline Week-1/Session2/Hometownwebpage.html:1]: This entire page needs formatting/Indentation. Use prettier in Vscode to format.

File: Week-1/Session2/Hometownwebpage.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    <ul>
+        <li>Visit Pandavleni Caves</li>
+        <li>Visit Sula Vineyards</li>
+        <li>Visit Trimbakeshwar Temple</li>
+        <li>Visit Kalaram Temple</li>
+        <li>Visit Someshwar Waterfall</li>
+    </ul>
+</section>
+
+</html>
```
Comment (sairamin) [inline Week-1/Session2/Hometownwebpage.html:50]: Please add a new line at the end of the file.

File: Week-1/Session2/Hometownwebpage.html
Comment (sairamin) [reply Week-1/Session2/Hometownwebpage.html:1]: Read about **Prettier** and **Eslint** and how to integrate them into your code editor

File: Week-1/Session2/Hometownwebpage.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+</head>
+
+<body>
+    <header>
+        <h1>Nashik, Maharashtra</h1>
+    </header>
+</body>
+
+
+<section>
```
Comment (sairamin) [inline Week-1/Session2/Hometownwebpage.html:15]: All these HTML tags should be inside body tag.

File: Week-1/Session2/PoemWeb Page.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<p>
```
Comment (sairamin) [inline Week-1/Session2/PoemWeb Page.html:2]: Please follow the same format everywhere.

**Pull request closed without merging**